mass_mailing_newsletter_welcome_mail/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Tecnativa - Cristina Martin R.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Welcome mail to new subscribers",
+    "summary": "Send an automated welcome mail to new newsletter subscribers",
+    'license': 'AGPL-3',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'category': 'Social',
+    'version': '11.0.1.0.0',
+    'website': 'https://github.com/OCA/website',
+    'depends': [
+        'mass_mailing',
+        'website',
+        'website_crm',
```

Comment:
Why `website` and `website_crm`? :thinking:

Reply:
Testing website forms, but it's not necessary that dependency. I remove it